shared/src/vyxal/Elements.scala

```diff
@@ -187,6 +179,14 @@ object Elements:
             case _ => throw InvalidListOverloadException("Ạ", b, "Number")
         if a.isInstanceOf[String] then temp.mkString
         else temp
+      case (a, b: VList, c) =>
+        var temp = ListHelpers.makeIterable(a)
+        for i <- ListHelpers.makeIterable(b) do
+          i match
+            case ind: VNum => temp = ListHelpers.assign(temp, ind, c)
+            case _ => throw InvalidListOverloadException("Ạ", b, "Number")
```

Comment:
```suggestion
        val temp = ListHelpers.makeIterable(b).foldLeft(ListHelpers.makeIterable(a)) {
          case (temp, ind: VNum) => ListHelpers.assign(temp, ind, c)
          case _ => throw InvalidListOverloadException("Ạ", b, "Number")
        }
```
No idea if it works but it probably does